scripts/templates/table/table.ejs

```diff
@@ -1,22 +1,34 @@
 <table>
-  <h1> </h1>
-<thead>
-  <tr>
-    <% for(var i=0; i<actors['title'].length; i++) {%>
-      <th><%= actors['title'][i]['name'] %></th>
-    <% } %>
-  </tr>
-</thead>
-<tbody>
-  <% for(var i=0; i<actors['data'].length; i++) {%>
-    <tr>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['municipalities'][0]['unit'] : ''  %>"><%= actors['data'][i]['municipalities'][0]['value'] %></td>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['trade'][0]['unit'] : ''  %>"><%= actors['data'][i]['trade'][0]['value']  %></td>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['deforastation'][0]['unit'] : ''  %>"><%= actors['data'][i]['deforastation'][0]['value']  %></td>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['water'][0]['unit']: ''  %>"><%= actors['data'][i]['water'][0]['value']  %></td>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['conflict'][0]['unit'] : ''  %>"><%= actors['data'][i]['conflict'][0]['value']  %></td>
-      <td data-unit="<%= i == 0 ? actors['data'][i]['fire'][0]['unit'] : ''  %>"><%= actors['data'][i]['fire'][0]['value']  %></td>
-    </tr>
+  <% if(type === 'table_header'){ %>
+    <thead>
+      <tr>
+        <% for(let i=0; i<actors['title']['value'].length; i++) { %>
+          <th><%= actors['title']['value'][i] %></th>
+        <% } %>
+      </tr>
+    </thead>
+  <% } %>
+  <% if(type === 'table_header'){ %>
+    <tbody>
+      <% for(let i=0; i<actors['data'][actors['title']['value'][0]].length; i++) { %>
+        <tr>
+        <%  for(let j=0; j<actors['title']['value'].length; j++){%>
+          <td data-unit="<%= i == 0 ? actors['title']['units'][j] : ''  %>">
+            <%= actors['data'][actors['title']['value'][j]][i] %>
+            <% if(j == 0){ %> <svg class="icon icon-check"><use xlink:href="#icon-outside-link"/></svg> <% } %>
+          </td>
+        <% } %>
+        </tr>
+      <% } %>
+    </tbody>
+  <% } else {%>
+    <tbody>
+      <% for(let i=0; i<actors.length; i++) { %>
+        <tr>
+          <td><%=i+1%>.<%= actors[i]['name'] %> <svg class="icon icon-check"><use xlink:href="#icon-outside-link"/></svg></td>
+          <td data-unit="<%= i == 0 ? '%' : ''  %>"><%= (actors[i]['value'] * 100).toFixed(2) %></td>
```

Comment:
`.toFixed(2)` <-- avoid logic in templates, please. Do this in the view, before render it.